jetty-core/jetty-security/src/main/java/org/eclipse/jetty/security/jaas/spi/AbstractLoginModule.java

```diff
@@ -35,7 +35,8 @@
  * AbstractLoginModule
  *
  * Abstract base class for all LoginModules. Subclasses should
- * just need to implement getUserInfo method.
+ * implement getUser method, and subclass JAASUser to implement
+ * doFetchRoles method.
```

Comment:
Please use {@link} to refer to the methods, so that users would be able to follow the links, and we will be sure that method renaming will automatically fix the javadocs.

Reply:
Done.